src/plugins/lua/LuaImGui.cpp

```diff
@@ -81,56 +81,19 @@ LuaImGui::~LuaImGui()
 {
 }
 
-class ScopedYieldDisabler
-{
-	bool m_origAllowYield = true;
-	std::shared_ptr<LuaThread> m_threadPtr;
-
-public:
-	ScopedYieldDisabler(const std::shared_ptr<LuaThread>& thread_ptr)
-		: m_threadPtr(thread_ptr)
-	{
-		if (m_threadPtr)
-		{
-			m_origAllowYield = m_threadPtr->GetAllowYield();
-			thread_ptr->SetAllowYield(false);
-		}
-	}
-
-	~ScopedYieldDisabler()
-	{
-		if (m_threadPtr)
-		{
-			m_threadPtr->SetAllowYield(m_origAllowYield);
-		}
-	}
-};
-
 bool LuaImGui::Pulse() const
 {
 	bool success = true;
 	try
 	{
-		int yield_count = 0;
-		constexpr int max_yields = 20;
 		ScopedYieldDisabler disableYield(LuaThread::get_from(m_thread.state()));
 
-		sol::protected_function_result result;
-		do
+		sol::function_result result = m_coroutine();
```

Comment:
This change from protected seems to result in lua stacktraces no longer printing on release builds, is that intentional? Still outputs fine on debug builds. On release, you just get the popup and message about how to resume the overlay but then no indication of the error that caused it.

Reply:
![image](https://user-images.githubusercontent.com/57729/235991839-e51a7737-5f3a-4931-bc12-100ad150bab6.png)
screenshot so you can see the highlighting.

Reply:
to translate, this change is a no-op

Reply:
ok yeah, looks like release builds change the SOL_SAFE_FUNCTION_OBJECTS_I_ define. thats not what i want..